Source/Core/Core/CoreTiming.cpp

```diff
@@ -101,8 +101,7 @@ void CoreTimingManager::Init()
   m_is_global_timer_sane = true;
 
   // Reset data used by the throttling system
-  m_throttle_last_cycle = 0;
-  m_throttle_deadline = Clock::now();
+  ResetThrottle(0);
```

Comment:
this should reset to m_globals.global_timer, which is zero, but i dont think that it should be passed in, because there is never a situation where the throttle clock should be set to anything different then the global timer when resetting.